intTests/test_examples/test.sh

```diff
@@ -1,14 +1,9 @@
 #!/bin/sh
 set -e
-
-mkdir -p tmp
-cp -r ../../examples tmp
-
+CRYPTOLPATH="../../deps/cryptol-specs"
```

Comment:
Hrm, this worked locally (and also broke locally when I set it to something else). Should I `export` it?